src/app/faq/faq.component.spec.ts

```diff
@@ -0,0 +1,25 @@
+import { ComponentFixture, TestBed } from '@angular/core/testing';
```

Comment:
this seems to be a duplicate file. rather than making a copy it would probably be best to consider this a global file and just have each instance call it in separately.